static/app/gettingStartedDocs/node/awslambda.tsx

```diff
@@ -10,71 +14,177 @@ import {
   getCrashReportModalConfigDescription,
   getCrashReportModalIntroduction,
 } from 'sentry/components/onboarding/gettingStartedDoc/utils/feedbackOnboarding';
+import {AwsLambdaArn} from 'sentry/gettingStartedDocs/node/awslambdaArnSelector';
 import {t, tct} from 'sentry/locale';
-import {trackAnalytics} from 'sentry/utils/analytics';
 import {
   getInstallConfig,
   getNodeProfilingOnboarding,
   getSdkInitSnippet,
 } from 'sentry/utils/gettingStartedDocs/node';
-import {
-  InstallationMode,
-  platformOptions,
-} from 'sentry/views/onboarding/integrationSetup';
+
+export enum ModuleFormat {
+  CJS = 'cjs',
+  ESM = 'esm',
+}
+
+export const platformOptions = {
+  moduleFormat: {
+    label: t('Module Format'),
+    items: [
+      {
+        label: t('Lambda Layer (CJS)'),
```

Comment:
l: Since this selection should focus on the module format, let's switcht his around, wdyt?

(I realize we have the same headings in docs; something for us to change/improve on!)

```suggestion
        label: t('CJS: Lambda Layer'),
```